aws-rds/README.md

```diff
@@ -24,81 +24,181 @@ Use SignalFx to monitor Amazon Relational Database Service (RDS) via [Amazon Web
 
-By default, SignalFx will import all CloudWatch metrics that are available in your account. To retrieve metrics for a subset of available services or regions, modify the connection on the Integrations page.
+#### If you are upgrading from version 0.1.0 (Python 2.7) to version 0.2.0 (Python 3.x)
+If you are upgrading to version 0.2.0, ensure that the AWS Lambda handler is set to `enhanced_rds.lambda_script.lambda_handler`.
 
-#### RDS ENHANCED MONITORING
+#### Encrypting your SignalFx access token
+The Lambda function uses your SignalFx access token to send metrics to SignalFx, as an environment variable to the function. While Lambda encrypts all environment variables at rest and decrypts them upon invocation, AWS recommends that all sensitive information such as access tokens be encrypted using a KMS key before function deployment, and decrypted at runtime within the code.
 
-SignalFx supports an integration with <a target="_blank" href="http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.html">RDS Enhanced Monitoring</a> using an AWS Lambda function, described below. This integration also includes built-in dashboards designed specifically for the metrics from Enhanced Monitoring. This Enhanced Monitoring integration complements the CloudWatch-based integration described above.  
+Both Serverless Application Repository and build from source deployment procedures below include instructions for using either an encrypted or non-encrypted access token.
 
-You can choose to deploy the function either from the Serverless Application Repository (recommended) or from source. Choose a deployment method and follow the steps below to encrypt your SignalFx access token, customize the metrics that will be sent to SignalFx, and create and deploy the new function. 
+## Deploying through the Serverless Application Repository
+Deploying through the Serverless Application Repository is a four-step process if you manually encrypt your access token, and a three-step process otherwise:
 
-Before you begin, you must enable the Enhanced Monitoring option for the RDS instances you want to monitor using this integration. <a target="_blank" href="http://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/USER_Monitoring.OS.html">Click here for instructions on enabling Enhanced Monitoring</a>.
+### 1. Set up an encryption key and encrypt your access token (if desired)
+Start with this step only if you chose to manually encrypt your access token.
+Either create a new KMS encryption key or select a preexisting one. **The key
+must be in the same availability zone as the RDS instances you are
+monitoring.** You can create and manage encryption keys from IAM in the AWS
+management console. Documentation about KMS encryption from the CLI can be found
+[here](http://docs.aws.amazon.com/cli/latest/reference/kms/encrypt.html). Make sure you have access to the cipher text output by the encryption as well as the key ID of the encryption key you used.
```

Comment:
Instead of "here," I would use the actual name of the link for the text. For ex, "See [encrypt] for more information about KMS encryption from the CLI."